src/containers/Ledger/ledger.scss

```diff
@@ -144,18 +180,20 @@
         }
 
         .ledger-cols {
-          display: table;
+          display: grid;
+          text-align: center;
+          grid-template-columns: 1fr 1fr;
+          gap: 25px;
           width: 100%;
           margin-bottom: 20px;
-          text-align: right;
+
+          @include for-size(desktop-up){
```

Comment:
> Because of the sheer number of metrics this needs to be revisited.
> 
> 
> 
> We thinking flex-box with wrap on and a gap of 48px.

I'll have to play around with css before agreeing to your point